Tests/ExperimentIntegrationTests/test_integration.py

```diff
@@ -0,0 +1,8 @@
+def test_testing(xcodebuild, setup_experiment, start_app):
```

Comment:
Where should we put the `.py` files? The `Tests/` folder contains only `.swift` files so far. The `SyncIntegrationTests` are in the root folder.

Reply:
I was told to move the folder here, see [here](https://github.com/mozilla-mobile/firefox-ios/pull/15574#issuecomment-1653891885)

Reply:
This looks good to me @clarmso The struture is the same as the SyncIntegrationTests folder, we have the new folder ExperimentIntegrationTests and the .py files in it.
At some point we should move the SyncIntegrationTests folder into Tests so that all test folder are in the same directory.